src/lib/Zikula/Bundle/CoreBundle/DependencyInjection/CoreExtension.php

```diff
@@ -48,6 +51,110 @@ public function load(array $configs, ContainerBuilder $container)
                 mkdir($cacheDir.'/'.$dir, 0777, true);
             }
         }
+        
+        //$this->registerTranslatorConfiguration($config['translator'], $container);
+        $this->registerTranslatorConfiguration($container);        
+        
+    }
+    
+    /**
+     * Loads the translator configuration.
+     *
+     * @param array            $config    A translator configuration array
+     * @param ContainerBuilder $container A ContainerBuilder instance
+     */
+    private function registerTranslatorConfiguration(ContainerBuilder $container)
+    {
+
+    	// Use the "real" translator instead of the identity default
+    	$container->setAlias('translator', 'translator.default');
+    	$translator = $container->findDefinition('translator.default');
+    	$translator->addMethodCall('setFallbackLocales', array(array('en')));
```

Comment:
Here this should be determined from configuration please take a look into https://github.com/symfony/FrameworkBundle/blob/2.6/DependencyInjection/FrameworkExtension.php#L636